runtime/src/accounts.rs

```diff
@@ -594,7 +594,10 @@ impl Accounts {
             }
         };
 
-        if payer_account.lamports() < fee + min_balance {
+        if payer_account.lamports() < fee.checked_add(min_balance).ok_or_else(|| {
```

Comment:
Saturating math probably not work here:  `u64::MAX < u64::MAX.saturating_add(1)` should be true but not; similar `1.saturating_sub(99)<0` should be true but also not.

Using `checked_add` instead, cause if `fee.checked_add(min_balance)` exceeds boundary, the payer surely won't be able to pay, even it has u64::MAX balance.

Reply:
yeah nice catch!